draft-ietf-lpwan-schc-over-lorawan.md

```diff
@@ -412,8 +412,11 @@ fragmentation sessions and are successively described in the next sections.
 
 ### DTag
 
-A LoRaWAN device cannot interleave several fragmented SCHC datagrams. This
-field is not used and its size is 0.
+A LoRaWAN device cannot interleave several fragmented SCHC datagrams on the same
+FPort.  This field is not used and its size is 0.
```

Comment:
I think it's very good, but maybe slightly more restrictive than it absolutely needs to be. If I am not mistaken, we should have only one session per ruleId (so per SCHC gateway). While we recommend using 8bit ruleId for fragmentation, which will fit perfectly in the fport, we don't absolutely mandate it. It could be therefore possible to have 2 different fragmentation ruleIds for two different SCHC gateways that are 9bits long for example and that share the same fport value. While I don't believe this would really be a big issue, I don't see a reason to restrict it if my reasoning is correct.

Reply:
Hi @ivajloip , true we should not restrict the ruleId on 8 bits. is the following proposition better for you ?

> A LoRaWAN device cannot interleave serveral fragmentedSCHC datagrams on one fragmentation RuleId set.
> Note: The device can still have several parallel fragmentation sessions with one or more SCHC gateway(s) thanks to distinct set of fragmentation RuleIds, cf {{rule-id-management}}

So there is no more implicit limitation of the RuleID length, but it feels harder to read to me

Reply:
I just read again the rule-id-management section and I think I understood why originally it was saying FPort - as we don't expect the NS to understand SCHC, different SCHC gateways will need to have at least different FPort values. If we want to permit having 2 fragmentation sessions on different RuleIds to the same SCHC gateway, we could write

```
For any given fragmentation RuleId, there could be at most one fragmentation session. No interleave of several fragmented SCHC datagrams using the same RuleId is permitted in uplink and no interleave is permitted in downlink direction.
Note: The device can still have several parallel fragmentation sessions with one or more SCHC gateway(s) thanks to the use of distinct fragmentation RuleIds and different FPorts if different SCHC gateways are used, cf {{rule-id-management}}
```

If not, as the minimum length of RuleId is 8 bits, and the first 8 bits need to be different for different SCHC gateways, I think the original text was correct - we need the FPorts to differ in order to know to which SCHC gateway a packet needs to go.